src/components/Global/Navbar.js

```diff
@@ -1,5 +1,5 @@
-import { Fragment } from "react";
 import Link from 'next/link';
+import { Fragment } from "react";
 import { Popover, Transition } from "@headlessui/react";
 import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
 import { SiGithubsponsors } from "react-icons/si";
```

Comment:
```suggestion
import { Fragment } from "react";
import { Popover, Transition } from "@headlessui/react";
import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
import { SiGithubsponsors } from "react-icons/si";
import Link from 'next/link';
```

Let's try to keep the git log to the minimum